app/core/components/ProposalCard/index.tsx

```diff
@@ -78,16 +77,16 @@ export const ProposalCard = (props: IProps) => {
                       {props.status}
                     </span>
                     <div className="ProposalCard__tier">
-                      <Link
-                        to="https://wizeline.atlassian.net/wiki/spaces/wiki/pages/3075342381/Innovation+Tiers"
+                      <StyledLink
```

Comment:
You can probably then use the MUI [Link](https://mui.com/material-ui/react-link/) component instead, which does just this.

Reply:
Thanks for the suggestion!
I updated the code to use the `Link` component from MUI